vendor/assets/stylesheets/dvl/components/navbar.scss

```diff
@@ -110,9 +110,8 @@
   text-align: center;
   display: inline-block;
   padding: 0 $rhythm / 2;
-  height: $rhythm * 2;
-  line-height: $rhythm * 2;
-  font-size: $fontSmallest;
+  height: $lineHeightSmallest;
+  @include fontSmallestBody;
```

Comment:
Here's how this badge looks in `master`:

<img width="249" alt="screen shot 2016-05-05 at 7 42 19 am" src="https://cloud.githubusercontent.com/assets/1270317/15046514/f0787e5c-1294-11e6-89c3-09b4cbad1150.png">

Here's how it looks here:

<img width="181" alt="screen shot 2016-05-05 at 7 42 40 am" src="https://cloud.githubusercontent.com/assets/1270317/15046516/f9989602-1294-11e6-9739-3c01b4feb064.png">

:'(